packages/components/app/styles/components/page-header.scss

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// PAGEHEADER
+//
+
+.hds-page-header {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  container-type: inline-size;
+}
+
+.hds-page-header__body {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem 1rem;
+  container-type: inline-size;
```

Comment:
I see there are two `container-type: inline-size` declaration (one here, and one in the parent). Are you sure you need this one here? If I comment it out in the devtools I don't see any effect 🤔

Reply:
Yes, looks like you're correct. Still figuring out the details of containers, but looks like declaring it once at the top of the component is sufficient.

Reply:
@jorytindall in case you didn't see it https://developer.mozilla.org/en-US/docs/Web/CSS/container-name (in case you need multiple "containers"; but in this case one is enough)